src/lib/strings.ts

```diff
@@ -22,3 +22,13 @@ export const transformToEllipsis = (text: string, maxLength: number) => {
 		return text.slice(0, maxLength - 3) + "...";
 	}
 };
+
+export const extractDockerImageKey = (string: string) => {
```

Comment:
I've started adding tests to the utility functions. We should add the tests whenever we create a new one